scripts/pylib/pytest-twister-harness/src/twister_harness/plugin.py

```diff
@@ -117,6 +117,10 @@ def pytest_addoption(parser: pytest.Parser):
         choices=('function', 'class', 'module', 'package', 'session'),
         help='The scope for which `dut` and `shell` fixtures are shared.'
     )
+    twister_harness_group.addoption(
+        '--fixture', action='append', dest='fixtures', metavar='FIXTURE', default=[],
```

Comment:
this might get confused with https://pytest.org/en/6.2.x/fixture.html :)

Reply:
Yeah, but it's not really a user-facing argument. It's automatically passed via twister.

Reply:
I agree that it can be confusing. `fixture` in pytest community has different meaning. Maybe inside this pytest plugin, better to use something like `--twister-fixture` or `--platform-fixture`? or at least to explain in help message, that this is not pytest fixture

Reply:
Sure, I can change the name of the argument, I think `--twister-fixture` is better. I'll update the help text as well.

Reply:
Argument renamed.